File: Command/C#/AccountProcess/Account.cs
namespace AccountProcess
{
    public class Account
    {
        public int Balance { get; set; }

        public void Process(Command c)
        {
            switch (c.TheAction)
            {
                case TypeOperation.Deposit:
                    Balance += c.Amount;
                    c.Success = true;
                    break;
                case TypeOperation.Withdraw:
                    c.Success = Balance >= c.Amount;
                    if (c.Success) Balance -= c.Amount;
                    break;
                default:
                    throw new ArgumentOutOfRangeException();
            }
        }
    }
}

File: Command/Python/AccountProcess/account.py
from action import Action
from command import Command


class Account:
    def __init__(self, balance: int = 0):
        self.balance = balance

    def process(self, command: Command):
        if command.action == Action.DEPOSIT:
            self.balance += command.amount
            command.success = True
        elif command.action == Action.WITHDRAW:
            command.success = self.balance >= command.amount
            if command.success:
                self.balance -= command.amount
